dotCMS/src/main/java/com/dotcms/rendering/velocity/services/StringifyObject.java

```diff
@@ -0,0 +1,104 @@
+package com.dotcms.rendering.velocity.services;
+
+import com.dotcms.repackage.com.ibm.icu.text.SimpleDateFormat;
+
+import java.io.StringWriter;
+import java.util.Collection;
+import java.util.Date;
+import java.util.Iterator;
+
+final class StringifyObject {
+    final String stringified;
+
+    public StringifyObject(final Object o) {
+
+        if(o instanceof String[]) {
+            this.stringified = stringifyObject((String[]) o);
+        }
+        else if(o instanceof Collection) {
+            this.stringified = stringifyObject((Collection) o);
+        }
+        else if(o instanceof Boolean) {
+            
+            this.stringified = stringifyObject((Boolean) o);
+        }
+        else if(o instanceof Date) {
+            
+            this.stringified = stringifyObject((Date) o);
+        }
+        else if(o instanceof String) {
+            
+            this.stringified = stringifyObject((String) o);
+        }
+        else {
+            this.stringified = o.toString();
+        }
+    }
+
+
+    private String stringifyObject(final String[] str) {
+        StringWriter sw = new StringWriter();
+        sw.append('[');
+        for (int i = 0; i < str.length; i++) {
+            sw.append('"')
+                .append(str[i])
+                .append("\"");
+            if (i != str.length - 1) {
+                sw.append(",");
+            }
+        }
+        sw.append(']');
+        return  sw.toString();
+    }
+
+    private String  stringifyObject(final Collection co) {
+        StringWriter sw = new StringWriter();
+
+        sw.append('[');
+        Iterator<Object> it = co.iterator();
+        while (it.hasNext()) {
+            Object obj = it.next();
+            sw.append('"')
+                .append(obj.toString())
+                .append("\"");
+            
+            if(it.hasNext()) {
+                sw.append(",");
+            }
+        }
+        sw.append(']');
+        return  sw.toString();
+    }
+
+    private String  stringifyObject(final Boolean o) {
+
+        return ((Boolean)o).toString();
+    }
+
+    private String  stringifyObject(final String x) {
+        StringWriter sw = new StringWriter();
+        if (x.startsWith("$")) {
+            return  sw.toString();
+        } else {
+            sw.append('"');
+            sw.append(x.toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "x" is already a string, there's no need to call "toString()" on it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1858)